src/angularComponent/ngcOmniboxFieldDirective.js

```diff
@@ -30,7 +30,7 @@ export default function ngcOmniboxFieldDirective($document) {
           '{{::omnibox.autofocus === \'true\' || undefined}}');
       input.setAttribute('ng-disabled', 'omnibox.ngDisabled()');
 
-      tElement.html(input.outerHTML);
+      tElement[0].appendChild(input);
```

Comment:
I don't actually know. My gut just thinks that re-using the same DOM object you've already created and manipulated would be faster than having the browser re-parse a brand new string to re-create another DOM object.